.github/workflows/native-cron-build.yml

```diff
@@ -29,8 +32,8 @@ jobs:
       - name: Checkout Quarkus
         uses: actions/checkout@v2
         with:
-          repository: quarkusio/quarkus
-          ref: master
+          repository: aguibert/quarkus
```

Comment:
We should probably revert all changes in this file?

Reply:
hi @machi1990, thanks for having a look at the PR. All the workflow changes right will be rolled back. I've hacked up the workflow files just to make the testing quicker

Reply:
Thanks, realised that I jumped quickly with the review and missed your comment about this being a WIP.